packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1175,7 +1175,7 @@ export class AccountHandler {
       scope = { contains: () => true };
     } else {
       uid = auth.credentials.user;
-      scope = ScopeSet.fromArray(auth.credentials.scope!);
+      scope = ScopeSet.fromArray(auth.credentials.scope || []);
```

Comment:
The `scope` exists here.  (See https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/routes/auth-schemes/auth-oauth.js if interested.)  Perhaps we can add a type like
```
// Use this for requests that have been auth'd with an OAuth access token.  If
// the scope isn't something ScopeSet.fromString can handle, an error would've
// occurred already.
export interface OauthAccessTokenAuthedRequest extends AuthRequest {
  auth: AuthRequest['auth'] & {
    credentials: AuthRequest['auth']['credentials'] & { scope: string[] };
  };
}
```
And use that instead of AuthRequest on line 1170.

Reply:
Any reason we can't update `AuthRequest`? Not familiar with where that's used but the name implies to my brain that it'd already have this property, no?

Reply:
Looks like there are no issues updating the `AuthRequest` type.